ClientApp/viking-vault/src/UserDashboard/TransferFunds/TransferFundsPage.tsx

```diff
@@ -0,0 +1,340 @@
+import React from 'react'
+import {constants, currencyEnum} from "../../Resources/Constants";
+import SideBar from '../../Common/SideBar';
+import TopBar from '../../Common/TopBar';
+import UserIcon from '../../Common/UserIcon';
+import './TransferFunds.css';
+import '../ExchangeForm.css';
+import TransferFundsModal from './TransferFundsModal';
+import Toggle from '../../Common/Toggle';
+import TransferRequests from '../TransferRequests';
+
+const transferUrl = `${constants.baseUrl}transferFunds`;
+const requestUrl = `${constants.baseUrl}transferRequests`;
+const currencyMap = {
+  [currencyEnum.ron]: 0,
```

Comment:
This mapping is a bit rendundant as TypeScript has enum types...make use of them instead

Reply:
We used this mapping across different parts of the code so I'd rather keep it consistent